packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 52.3.0 - June 30, 2021
```

Comment:
Removed some under-the-hood changes or things that are going into the new conflict detection mechanism that is not yet exposed